backend/voice_service.py

```
@@ -0,0 +1,332 @@
+                'original_text': text,
+                'error': None
+            }
+            
+        except Exception as e:
+            logger.error(f"Error translating text: {e}", exc_info=True)
+            return {
+                'translated_text': None,
+                'source_language': None,
+                'target_language': None,
+                'original_text': text,
+                'error': f'Translation error: {str(e)}'
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)